plugin.program.AML/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.program.AML" name="Advanced MAME Launcher" version="0.9.11" provider-name="Wintermute0110">
+<addon id="plugin.program.AML" name="Advanced MAME Launcher" version="0.9.12" provider-name="Wintermute0110">
```

Comment:
would be cool if you could adjust the `license` element to use a spdx identifier: https://spdx.org/licenses/

Reply:
Thank for the advice, I already changed it in master. By the way, the example given in [Kodi Wiki Addon](https://kodi.wiki/view/Addon.xml) in the `<license>` is deprecated according to the list you posted and must be updated.